tests/test_virtualenv.py

```diff
@@ -23,6 +23,7 @@
 
 
 IS_WINDOWS = nox.virtualenv._SYSTEM == "Windows"
+RAISE_ERROR = "RAISE_ERROR"
```

Comment:
Very srs bznz!

Reply:
Yes. I'm exceptionally proud of this line. :) Well, I at least hope it's more explicit than some fake path string like `"c:\this\breaks\python.exe"`.